lib/collections/addon/components/collections-submission/template.hbs

```diff
@@ -146,11 +149,11 @@
                     {{t (concat this.i18nKeyPrefix 'cancel')}}
                 </button>
                 {{#if this.edit}}
-                    <button class='btn btn-success' {{action (perform this.save)}}>
+                    <button data-test-collection-public-node-submit data-analytics-name='Update' class='btn btn-success' {{action (perform this.save)}}>
                         {{t (concat this.i18nKeyPrefix 'update' '_button')}}
                     </button>
                 {{else}}
-                    <button class='btn btn-success' {{action this.setShowSubmitModal}} disabled={{not (eq this.activeSection this.sections.submit)}}>
+                    <button data-test-collection-private-node-submit data-analytics-name='Submit' class='btn btn-success' {{action this.setShowSubmitModal}} disabled={{not (eq this.activeSection this.sections.submit)}}>
```

Comment:
I don't understand why you've use `private-node` in the test selector name here (and `public-node` above).